<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QMainWindow" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QListWidget" name="listWidget">
       <item>
        <property name="text">
         <string>Joystick</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Serial Port</string>
        </property>
       </item>
      </widget>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="JoystickSettingsPage" name="joystickPage"/>
       <widget class="SerialPortSettingsPage" name="serialPage"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SerialPortSettingsPage</class>
   <extends>QWidget</extends>
   <header>serialportsettingspage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>JoystickSettingsPage</class>
   <extends>QWidget</extends>
   <header>joysticksettingspage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
